code/plugins/api/articles/articles/helper/contenthelper.php

```
@@ -0,0 +1,209 @@
+
+		$params = array('style' => $style);
+		ob_start();
+
+		echo $renderer->render($mod, $params);
+
+		$this->mods[$module] = ob_get_clean();
+
+		return $this->mods[$module];
+	}
+	
+	public function mapCategory( $cat )
```

PHPMD (2.15.0)

Avoid unused parameters such as '$cat'. (Unused Code Rules)

(UnusedFormalParameter)